cmd/gossamer/configcmd.go

```diff
@@ -72,29 +70,29 @@ func makeNode(ctx *cli.Context) (*dot.Dot, *cfg.Config, error) {
 	}
 
 	// Trie, runtime: load most recent state from DB, load runtime code from trie and create runtime executor
-	r, err := loadStateAndRuntime(stateSrv.Storage, ks)
+	rt, err := loadStateAndRuntime(stateSrv.Storage, ks)
 	if err != nil {
 		return nil, nil, fmt.Errorf("error loading state and runtime: %s", err)
 	}
 
-	// load extra genesis data from DB
-	gendata, err := stateSrv.Storage.LoadGenesisData()
+	// load genesis from JSON file
+	gen, err := genesis.LoadGenesisFromJSON(getGenesisPath(ctx))
```

Comment:
we shouldn't be loading anything from the genesis file here, it should all happen when `gossamer init` is done

Reply:
Hmm... I did this because the bootnodes are string slice type when using `Genesis` and we are currently unable to use SCALE encoding with a byte slice. All we need for p2p service here is `Name` and `Bootnodes`. Why should we not be using the genesis file here? Can we combine `init` with action?

Reply:
I believe the SCALE issue has been fixed, unless you are talking about something else? 
yes we need Name and Bootnodes, which should already have been stored in the DB from doing init. the idea is that we don't need to pass in the genesis file when running just `gossamer`.

Reply:
If we do not want to load from genesis file here, we can solve #539 first. I thought using `Genesis` here was an improvement but either way works for me.

Reply:
okay yeah I'm gonna say do #539 first, is there a reason you can't retrieve it from the DB? I guess because decoding only decodes 1 bootnode?

Reply:
correct, the length is currently fixed with a length of `1`